README.md

```diff
@@ -59,6 +59,88 @@ and health-checking mechanisms.
   sudo yum -y install consul-1.9.1
   ```
 
+* Add yum repos:
+ 
+  * Cortx_iso.repo
+  ```sh
+  vi /etc/yum.repos.d/cortx_iso.repo
+  ```
+  * Paste:
+  ```sh
+  [cortx_iso]
+  baseurl=file:///var/artifacts/0//cortx_iso
+  gpgcheck=0
+  name=Repository cortx_iso
+  enabled=1
+  ```
+  * C7.8.2003 repo
+  ```sh
+  vi /etc/yum.repos.d/C7.8.2003.repo
+  ```
+  * Paste:
+  ```sh
+  # C7.8.2003
+  [C7.8.2003-base]
```

Comment:
Why do we need this repo specifically for hare? looks like a generic one, so ideally I think it should be mentioned as part of cortx README.